src/biome/text/pipeline.py

```diff
@@ -354,60 +360,62 @@ def explore(
         return explore_df
 
     def serve(self, port: int = 9998):
-        """Launches a REST prediction service with current model in a specified port (default is `9998)
+        """Launches a REST prediction service with the current model on a specified port (default is `9998)
```

Comment:
missing `

Reply:
moved the default to the Parameters section